week-6/Homework/mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,85 @@
 // Write your code here
+
+let slideIndex = 1;
+
+function plusSlide(n) {
+  clearInterval(intervalId);
+  slideShow((slideIndex += n));
+}
+
+function slideShow(n) {
+  let i;
+  let slides = document.getElementsByClassName("slideshow");
+  if (n > slides.length) {
+    slideIndex = 1;
+  }
+  if (n < 1) {
+    slideIndex = slides.length;
+  }
+  for (i = 0; i < slides.length; i++) {
+    slides[i].style.display = "none";
+  }
+  slides[slideIndex - 1].style.display = "block";
+}
+slideShow(slideIndex);
+let intervalId;
+let secondsForInterval = 2000;
+document.getElementById("inputBtn").addEventListener("click", enterSpeed);
+
+function enterSpeed() {
+  clearInterval(intervalId);
+  let inputValue = document.getElementById("input").value;
+  if (inputValue == "") {
+    alert("You didn't enter any value!");
+    let pauseBtn = document.getElementById("Pause");
+    let current = document.getElementsByClassName("active");
+    current[0].className = current[0].className.replace(" active", "");
+    pauseBtn.className += " active";
```

Comment:
Next time give a try to https://developer.mozilla.org/en-US/docs/Web/API/element/classList as it allows you to manipulate the classes of an element much easily and safely.